<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1657885581193' stop='1657885636031'>
    <name>Create wallet Sol</name>
    <title>Create wallet Sol</title>
    <test-cases>
        <test-case start='1657885581194' status='passed' stop='1657885607245'>
            <name>CS1: Verify Invalid wallet name while Sol account create</name>
            <title>CS1: Verify Invalid wallet name while Sol account create</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-0'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='device' value='Moto g71 11-11'/>
            </parameters>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1657885607246' status='broken' stop='1657885636028'>
            <name>CS2: Verify Setpin confirm pin mismatch while come popup handling</name>
            <title>CS2: Verify Setpin confirm pin mismatch while come popup handling</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-0'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='device' value='Moto g71 11-11'/>
            </parameters>
            <steps/>
            <attachments/>
            <failure>
                <message>An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details</message>
                <stack-trace>unknown error: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
                    at getErrorFromResponseBody (C:\Users\Guna\kanalabs-test\node_modules\webdriver\build\utils.js:198:12)
                    at NodeJSRequest._request (C:\Users\Guna\kanalabs-test\node_modules\webdriver\build\request\index.js:166:60)
                    at async Browser.wrapCommandFn (C:\Users\Guna\kanalabs-test\node_modules\@wdio\utils\build\shim.js:137:29)
                    at async Browser.$ (C:\Users\Guna\kanalabs-test\node_modules\webdriverio\build\commands\browser\$.js:91:17)
                    at async Browser.wrapCommandFn (C:\Users\Guna\kanalabs-test\node_modules\@wdio\utils\build\shim.js:137:29)
                    at async CreateWalletPage.enterConfirmPin (C:\Users\Guna\kanalabs-test\test\pageobjects\sol.pages.js\createwallet.page.js:93:16)
                    at async Context.&lt;anonymous> (C:\Users\Guna\kanalabs-test\test\specs\createwallet.js:49:9)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>